source/_faq/unique_id.markdown

```diff
@@ -4,10 +4,21 @@ description: "This entity does not have a unique ID?"
 ha_category: Configuration
 ---
 
-When you see
+If you try to access the configuration dialog for an entity in your Home Assistant, you might end up seeing this message:
 
-```text
-This entity does not have a unique ID, therefore its settings cannot be managed from the UI
-```
+<p class='img'>
+<img src='/images/faq/faq_no_unique_id.jpg' alt='Screenshot of popup for no unique ID'>
+</p>
 
-this simply means that you can't change the `entity_id` or the friendly name from the UI. You can still use the _Customizations_ menu to customize that entity. Typically you'll see this when you create entities using YAML.
+This simply means that this entity does not have a unique identification e.g., a serial number or another identifier that is guaranteed to be static and never change\*. As a result, the normal editing process that allows you to change various settings through the user interface (such as the entity ID, icon, friendly name, etc.) is not possible here. 
+
+Typically, you'll see this when you create entities manually using YAML, but it can also appear if the integration that provides this entity simply cannot determine a unique ID. This however is not an error, but just a limitation of the integration you use.
+
+|                 | Unique ID                         | Entity ID                                                                                                                                                                                                                                                                                       |
```

Comment:
Is a table the right choice here? It's a lot of text to put in a table, especially on a small screen.

Reply:
What would be the alternative? I think a table is best to show this comparison, even if on small screen you might have to scroll. A regular text paragraph feels more verbose plus with a table the page feels easier to grasp (more visual structure).

Reply:
Have you previewed the actual result?

![image](https://user-images.githubusercontent.com/195327/105703225-31d22000-5f0d-11eb-8c6f-f7d2603b4d07.png)


![image](https://user-images.githubusercontent.com/195327/105703264-3f87a580-5f0d-11eb-81fa-a8c44a6cf6ab.png)


I agree with your general statement, but the reality is that it is now unreadable imho

Reply:
How about this? 

![image](https://user-images.githubusercontent.com/114137/105715620-1de2ea00-5f1e-11eb-93f8-3050da89a772.png)

![image](https://user-images.githubusercontent.com/114137/105715695-3521d780-5f1e-11eb-870a-6178b43d9ba3.png)